docs/book/programs/lsm.md

```diff
@@ -84,8 +84,12 @@ the kernel parameters:
 GRUB_CMDLINE_LINUX="lsm=[YOUR CURRENTLY ENABLED LSMs],bpf"
 ```
 
-Then rebuilding the grub configuration with:
+Then rebuilding the grub configuration with:\
+Use any of the commands listed below depending on your system to rebuild the grub.
```

Comment:
This doesn't look correct. Please merge these sentences into one sentence, which fits into the rest of the doc, for example:

```suggestion
Then rebuilding the grub configuration with any of the commands listed below (each of them might be available or not in different Linux distributions):
```

Reply:
@vadorovsky Thank You for your suggestions. I updated the doc.